workflows/expectant-parents-with-kiss-with-hailuo/README.md

```diff
@@ -1,10 +1,10 @@
-# Expectant Parents + Kiss (Hailuo)
+# AI Expectant Parents Kiss with Hailuo AI
```

Comment:
there is still a naming problem here, as if families are ‘getting close’ with hailuo instead of with each other, and what is expectant parents meaning